phys2bids/interfaces/txt.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+phys2bids interface for txt files.
+"""
+from bioread import read_file
+
+from phys2bids.physio_obj import BlueprintInput
+
+import numpy as np
+# '/home/vicente/phys2bids/phys2bids/tests/data/Test_multiscan100Hz_trig_samefreq_header.txt'
+def populate_phys_input(filename, chtrig):
+    """
+    Populate object phys_input
+    for now we this works with labchart files
+    """
```

Comment:
It would be great if you could add Inputs and Outputs to complete the docstring (see `phys_obj.py`).
Then we can use it for `acq.py`.

Reply:
I know this is picky, but can you check the indentation of the docstring?